### PR TITLE
Error fixes and enhancements

### PR DESCRIPTION
**Pull request by bgodlin**
- Fixed mustisig error
- Added asset registration handling
- Added error reasons (Closes sora-xor/sora-subquery#19)

### CODE REVIEW

File: schema.graphql [hunk 1]
```diff
@@ -26,13 +26,15 @@ type HistoryElement @entity {
   method: String!
   address: String! @index
   networkFee: String!
-  success: Boolean!
+  execution: ExecutionResult!
```
Comment (pgolovkin) [inline schema.graphql:29]: @bgodlin it's a significant change that may break mobile applications. Let's discuss it on the mobile meeting.

File: src/mappings/batchTransactionsHandler.ts [hunk 2]
```diff
@@ -31,6 +31,8 @@ export async function batchTransactionsHandler(extrinsic: SubstrateExtrinsic): P
 
     logger.debug("Caught batch transaction extrinsic")
 
+    // TODO Depending on the result of the execution, divide the logic of the stored data (just like in other handlers)
```
Comment (pgolovkin) [inline src/mappings/batchTransactionsHandler.ts:34]: todo is here

Comment (bgodlin) [reply src/mappings/batchTransactionsHandler.ts:34]: This improvement requires quite significant amount of time. I will remove the comment for now.

**Review by pgolovkin — APPROVED** [review]
Great job!

**Pull request merged**